### PR TITLE
Create sermon-on-the-letting-in-and-not-letting-in_ernst_waldschmidt.md

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/articles/sermon-on-the-letting-in-and-not-letting-in_ernst_waldschmidt.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Mahāmaudgalyāyana's Sermon on the Letting-in and Not Letting-in (of Sensitive Influences)"
+authors:
+  - "Ernst Waldschmidt"
+external_url: "https://journals.ub.uni-heidelberg.de/index.php/jiabs/article/view/8453/2360"
+drive_links:
+  - "https://drive.google.com/file/d/1sVIf327n20H0n_asX3wD4jPPYhJisvXS/view?usp=sharing"
+course: problems
+tags:
+  - sense-restraint
```
Comment (buddhist-uni) [inline _content/articles/sermon-on-the-letting-in-and-not-letting-in_ernst_waldschmidt.md:10]: let's add the `sa` tag for the Sumyakta Agama (this sutra is sa1176) and `manuscripts` as well

File: _content/articles/sermon-on-the-letting-in-and-not-letting-in_ernst_waldschmidt.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+year: 1978
+journal: jiabs
+publisher: Heidelberg University
```
Comment (buddhist-uni) [inline _content/articles/sermon-on-the-letting-in-and-not-letting-in_ernst_waldschmidt.md:13]: full names in quotes

File: _content/articles/sermon-on-the-letting-in-and-not-letting-in_ernst_waldschmidt.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+address: "Heidelberg, Germany"
+volume: 1
+number: 1
+pages: "25--33"
+---
+
+> "...if a monk avoids positive or negative inclinations when using his senses, Mara would not get access, would not get a hold."
```
Comment (buddhist-uni) [inline _content/articles/sermon-on-the-letting-in-and-not-letting-in_ernst_waldschmidt.md:20]: The `> ` will automatically add the quote marks

File: _content/articles/sermon-on-the-letting-in-and-not-letting-in_ernst_waldschmidt.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+A brief description of Mahāmaudgalyāyana's sermon on sense-restraint. The author then shares and translates Sanskrit fragments of this sermon found in Turfan (Turpan).
```
Comment (buddhist-uni) [inline _content/articles/sermon-on-the-letting-in-and-not-letting-in_ernst_waldschmidt.md:22]: Let's add a link to its Pāli Parallel: SN 35.243 at the end of the first sentence.

**Review by buddhist-uni — COMMENTED** [review]
In the file name it should end in `_lastname-firstname.md`

**Comment by JJM317** [comment]
Not finding SN 35.243 in the canon section of Github or on the website.
Should we create an entry for that first? Am I searching correctly? Also, I
wouldn't mind making the cross reference links in our meeting one more
time.

Thanks much,
~Joe

**Comment by buddhist-uni** [comment]
Just added it for you here: https://github.com/buddhist-uni/buddhist-uni.github.io/blob/main/_content/canon/sn35.243.md?plain=1  😁

**Pull request merged**